projects/managed-hubs/support.md

```diff
@@ -1,6 +1,112 @@
+
+:::{note}
+We should try to respond to all support-related communications **within one working day**.
+Even if the support request is not resolved, others appreciate when we keep them in-the-loop about what we're doing.
+:::
+
+## Tracking support items
+
+### FreshDesk requests
+
+For bugs, hub changes, improvements, and general questions, we use the [e-mail address `support@2i2c.org`](mailto:support@2i2c.org).
+This is connected with [the 2i2c FreshDesk account](https://2i2c.freshdesk.com/), and is a central place for all support-related requests.
+
+Many requests may be immediately resolved via a quick action or response to the community representative.
+However, if a request requires consultation with the team, first open a thread in the {guilabel}`#support-freshdesk` Slack channel, and if necessary open a backlog issue or a .
+
+(support:issues)=
+### Support issues
+
+If resolving a support issue requires ongoing or concerted work, open a backlog issue tagged with the {guilabel}`support` label.
+
+Support backlog items should:
+
+- Be placed in our [Sprint Board](coordination:sprint-board).
+- Include a reference to any FreshDesk tickets if they exist.
+- Be prioritized over other backlog items for that sprint.
+
+## Communication channels
+
+We use two Slack channels to discuss support-related questions on the 2i2c hub:
+
+- The [#support-freshdesk](https://2i2c.slack.com/archives/C028WU9PFBN) is for discussing support tickets on Freshdesk.
+  Open a Slack thread for each support ticket, to have conversation around it amongst the team.
+  This channel is private because there’s an assumption of privacy when we discuss community-specific support matters.
+- The [#hub-support](https://2i2c.slack.com/archives/C01DB2JRP8W) channel is for discussing more general support-related issues relating to operating the 2i2c Hubs, and not not related to a specific FreshDesk ticket.
+  This channel is public, so can be useful if we wish to invite conversation from non 2i2c team members.
+
+## Lifecycle of a support request
+
+
+Below is a summary of the major steps that make up the support process, from beginning to end of a ticket.
+
+:::{note}
+We should try to respond to all support-related communications within one working day.
+:::
+
+- When you receive a ticket, communicate that you have received it and note the next steps you’ll follow to resolve it (even if this just means investigating because you're not sure what is wrong).
+- If needed, open a [support issue and add it to the backlog](support:issues).
+- Identify an engineering team member that can pick up this work.
+- Give the user updates every day or two as we work to resolve the issue.
+- When it is resolved, tell the user what you’ve done to resolve the issue.
+- Close any FreshDesk tickets and support issues that are related to this request.
+
+## How to prioritize support requests
+
+- Ask whether an issue is "critical" (AKA does the hub not work at all) and prioritize these if so
```

Comment:
You can't ask if the issue is critical, IMHO.
People will tend to say, yes, it is critical... we should gather as much info as we can about the issue and decide if it is actually critical or not.

Reply:
Good point - I think here I actually meant more like "Ask yourself" rather than "Ask the other person" - will clarify this.